packages/explorer-helpers/src/readMultipartWebStream.ts

```diff
@@ -0,0 +1,174 @@
+import { Observable } from 'zen-observable-ts';
+import type { GraphQLError } from 'graphql';
+import type MIMEType from 'whatwg-mimetype';
+import type { ResponseData } from './postMessageRelayHelpers';
```

Comment:
I don't think this file exists as a sibling? might need to bring that type into this file?

Reply:
😵‍💫 oops, its in the `types.ts` file my bad!